libcontainer/rootfs_linux.go

```diff
@@ -748,6 +748,41 @@ func pivotRoot(rootfs string) error {
 }
 
 func msMoveRoot(rootfs string) error {
+	mountinfos, err := mount.GetMounts()
+	if err != nil {
+		return err
+	}
+
+	absRootfs, err := filepath.Abs(rootfs)
+	if err != nil {
+		return err
+	}
+
+	for _, info := range mountinfos {
+		p, err := filepath.Abs(info.Mountpoint)
```

Comment:
not needed either as paths returned from the kernel are absolute and clean (unless I'm missing something).

I have addressed this in https://github.com/opencontainers/runc/pull/2255 (a separate commit to keep things simple), PTAL @giuseppe